### PR TITLE
Playlist videoplayer

### PR DESCRIPTION
**Pull request by Sarahshr**
Add video player in playlist

### CODE REVIEW

**Review by Fx-Colliot — DISMISSED** [review]
good for me

**Review by ThomasMlh — DISMISSED** [review]
Everything seems ok

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -79,29 +80,16 @@ const Cgu = styled.a`
   }
 `;
 
-const Footer = () => {
+const Footer = ({ idVideo, idVid }) => {
+  console.log(idVideo);
```
Comment (LoicBrassart) [inline src/components/Footer.jsx]: Nope

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Remove the console.log

**Review by ThomasMlh — APPROVED** [review]
It's ok !

**Review by LoicBrassart — APPROVED** [review]
NmE

**Pull request merged**